.buildkite/release-builds.yml

```diff
@@ -17,20 +17,23 @@ steps:
 
   - label: ":wordpress: :testflight: WordPress Release Build (App Store Connect)"
     command: ".buildkite/commands/release-build-wordpress.sh $BETA_RELEASE"
+    priority: 1
     env: *common_env
     plugins: *common_plugins
     notify:
     - slack: "#build-and-ship"
 
   - label: ":wordpress: :appcenter: WordPress Release Build (App Center)"
     command: ".buildkite/commands/release-build-wordpress-internal.sh"
+    priority: 1
     env: *common_env
     plugins: *common_plugins
     notify:
     - slack: "#build-and-ship"
```

Comment:
Not related to this PR but seeing this pipeline as a diff made me wonder: why do we have AppCenter build for WordPress but not for Jetpack?

Should duplicate this step for Jetpack, just like we have for the TestFlight one? 🤔

Reply:
Agreed, I think that step should be duplicated for Jetpack. I can do that in another PR since it requires adding a separate script and Fastfile work.

Reply:
Actually I just realized that this might entail a bit more than just the step duplication, because this App Center build for WordPress is using the "WordPress Internal" scheme… and I don't think a corresponding "Internal build" scheme exists for Jetpack…

So either that would mean creating such a scheme… or be ok with only WordPress having Internal builds… but also, maybe that's a sign that the "WordPress-Internal" build is not so useful nor used by anyone after all…

Now that I think about it, I think @mokagio suggested a while ago that we drop those App Center Internal builds for WP… 🤔  Maybe it's time?

Reply:
> Now that I think about it, I think @mokagio suggested a while ago that we drop those App Center Internal builds for WP… 🤔 Maybe it's time?

Ah, this was it: paaHJt-33b-p2